src/MailThief.php

```diff
@@ -31,20 +31,26 @@ public function hijack()
     {
         Mail::swap($this);
         app()->instance(Mailer::class, $this);
+
+        $from = app()['config']['mail.from'];
+
+        if (is_array($from) && isset($from['address'])) {
+            $this->alwaysFrom($from['address'], $from['name']);
+        }
     }
 
     public function raw($text, $callback)
     {
         $message = Message::fromRaw($text);
-        $callback($message);
+        $message = $this->prepareMessage($message, $callback);
```

Comment:
I don't *think* we need to assign this back to the message because of how passing objects works. Can you try it out just dropping the `$message` assignment?